it/MobileTokenProxyISpec.scala

```diff
@@ -50,11 +47,19 @@ class MobileTokenProxyISpec
     }
   }
 
+  "GET /mobile-token-proxy/oauth/invalidService" should {
+    "return an invalid service error" in {
+      val response = wsUrl(
+        s"/mobile-token-proxy/oauth/invalidService/authorize?journeyId=dd1ebd2e-7156-47c7-842b-8308099c5e75"
+      ).get().futureValue
+      response.status shouldBe 500
+    }
+  }
+
   val test = Table(
     ("Name of Group Of Tests", "urlPrefix", "serviceId"),
     ("Old Url with no Service Id", "/mobile-token-proxy/oauth/", "ngc"),
-    ("New Url with NGC Service Id", "/mobile-token-proxy/oauth/ngc/", "ngc"),
-    ("New Url with RDS Service Id", "/mobile-token-proxy/oauth/rds/", "rds")
+    ("New Url with NGC Service Id", "/mobile-token-proxy/oauth/ngc/", "ngc")
```

Comment:
Once the parameter in `oauthRedirectSuccess` function has been removed, then you can tidy up the test table as `serviceId` field is no longer needed.